src/client/pfs/pfs.proto

```diff
@@ -121,6 +121,8 @@ message CommitInfo {
   // will also be nil) or is the output commit of a failed job (in which case
   // 'finished' will have a value -- the end time of the job)
   Object tree = 7;
+  repeated Object trees = 13;
+  Object datums = 14;
```

Comment:
Can we get comments on these about how these fields relate to each other? Can you have `tree` and `trees`? Or is `tree` effectively deprecated at this point?

Reply:
Input repo commits will have `tree` set and output repo commits with have `trees` set. Whether `tree` gets deprecated depends on how we change input repos moving forward. I guess I can add a comment there that clarifies that.